programs/ssl/ssl_client2.c

```diff
@@ -1205,6 +1215,8 @@ int main(int argc, char *argv[])
                     break;
                 default: goto usage;
             }
+        } else if (strcmp(p, "early_data_file") == 0) {
+            opt.early_data_file = q;
```

Comment:
Nitpick comments. move fopen here
```suggestion
        } else if (strcmp(p, "early_data_file") == 0) {
            opt.early_data_file = q;
            if ((early_data_fp = fopen(opt.early_data_file, "rb")) == NULL) {
                mbedtls_printf("failed\n  ! Cannot open '%s' for reading.\n",
                               opt.early_data_file);
                ret = MBEDTLS_ERR_SSL_INTERNAL_ERROR;
                goto exit;
            }

```